config/webpack/environment.js

```diff
@@ -10,6 +10,16 @@ const resolve = {
 environment.splitChunks();
 environment.config.merge({ resolve });
 
+// Excluding node_modules From Being Transpiled By Babel-Loader
+// One change to take into consideration, is that Webpacker 4 transpiles the node_modules folder with the babel-loader. This folder used to be ignored by Webpacker 3. The new behavior helps in case some library contains ES6 code, but in some cases it can lead to issues. To avoid running babel-loader in the node_modules folder, replicating the same behavior as Webpacker 3, we added the following code:
```

Comment:
```suggestion
// One change to take into consideration, is that Webpacker 4 transpiles the node_modules
// folder with the babel-loader. This folder used to be ignored by Webpacker 3.
// The new behavior helps in case some library contains ES6 code, but in some cases it can
// lead to issues.
// Avoid running babel-loader on the node_modules folder (thus replicating the same behavior
// as Webpacker 3).
```

Reply:
j'ai reformaté sur plusieurs lignes, mais sinon nickel, merci pour le commentaire et l'explication ❤️